README.md

```diff
@@ -420,6 +433,13 @@ Style: 0 / 0
 Total: 100 / 100
 ```
 
+We can target a specific submission ID using `--target-submission`.
+You can [get a history of all past submissions](#getting-a-history-of-all-past-submissions) to see a student's previous submission IDs and scores.
+Extending the previous example with a target submission ID of `123456789`:
```

Comment:
There needs to be a transition here.
Like: "For example, we can extend the previous example ...".